dashboard-api/docker-compose-common.yaml

```diff
@@ -0,0 +1,22 @@
+services:
+  cockroachdb:
+    image: cockroachdb/cockroach:v21.2.3
+    entrypoint: cockroach start-single-node --insecure --advertise-addr localhost
+    ports:
+      - "26257:26257"
+      - "8080:8080"
+    networks:
+      - dashboard-api
+  setup_cockroachdb:
+    image: cockroachdb/cockroach:v21.2.3
```

Comment:
Unless we are running a different version of cockroach on this project, at least on fly we are using v20.1.4, for example, https://github.com/vlab-research/fly/blob/b0e949944028edab871b895da10da49f02fb900e/devops/dev-cluster.sh#L31

Reply:
As we discussed in the other comment I'll leave this as it is for now :)